libs/key-management/src/key.service.spec.ts

```diff
@@ -381,6 +367,19 @@ describe("keyService", () => {
     });
   });
 
+  describe("makeSendKey", () => {
+    const mockRandomBytes = new Uint8Array(64) as CsprngArray;
```

Comment:
```suggestion
    const mockRandomBytes = new Uint8Array(16) as CsprngArray;
```

The input here is a 16 byte seed (even if it's unfortunately called "key" in the send model and service). 

Ref:
https://github.com/bitwarden/clients/blob/23f4af698498f2c570b03f9e9a390cbe5814c0dc/libs/common/src/tools/send/models/domain/send.ts#L84
https://github.com/bitwarden/clients/blob/23f4af698498f2c570b03f9e9a390cbe5814c0dc/libs/common/src/tools/send/services/send.service.ts#L68

Reply:
We should probably replace the Cspringarray type here and foremost document it better because this is very unobvious.